fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanInstallmentChargeUtil.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.portfolio.loanaccount.domain;
+
+import java.math.BigDecimal;
+import org.apache.fineract.organisation.monetary.domain.MonetaryCurrency;
+import org.apache.fineract.organisation.monetary.domain.Money;
+import org.springframework.stereotype.Service;
+
+@Service
+public class LoanInstallmentChargeUtil {
+
+    public Money waive(final LoanInstallmentCharge loanInstallmentCharge, final MonetaryCurrency currency) {
```

Comment:
I don't get just from the method name what "Money" it would return. Of course I might realize it's the waived amount but why make that assumption?

Since this does have side effects, let's just make it a void and make sure the caller asks for the waived amount after the method call. Much cleaner and readable.